lib/slax/poker.ex

```diff
@@ -63,15 +63,33 @@ defmodule Slax.Poker do
 
   def decide(round, score) do
     {org, repo, issue} = Github.parse_repo_org_issue(round.issue)
-
     client = Tentacat.Client.new(%{access_token: Github.api_token()})
 
-    case Tentacat.Issues.update(client, org, repo, issue, %{labels: ["Points: #{score}"]}) do
-      {200, _issue, _http_response} ->
+    with {200, labels, _http_response} <- Tentacat.Issues.Labels.list(client, org, repo, issue),
+         :ok <- maybe_remove_label(client, org, repo, issue, labels),
+         {200, _issue, _http_response} <-
+           Tentacat.Issues.Labels.add(client, org, repo, issue, ["Points: #{score}"]) do
+      :ok
+    else
+      {_response_code, %{"message" => error_message}, _http_response} ->
+        {:error, error_message}
+    end
+  end
+
+  defp maybe_remove_label(client, org, repo, issue, labels) do
```

Comment:
I think there are cleaner / more readable ways to express this same logic... one idea would be to have a separate function like `find_points_label(labels)` and call it up in your main `decide` function. Then you can have a `maybe_remove_label(client, org, repo, issue, nil), do: :ok` clause that doesn't do anything and a `maybe_remove_label(client, org, repo, issue, label) do...` function that actually does the work. That one could just do a case statement on the tentacat response.